src/euroeval/benchmark_modules/litellm.py

```diff
@@ -1093,3 +1316,23 @@ def try_download_ollama_model(model_id: str) -> bool:
             level=logging.DEBUG,
         )
         return True
+
+def safe_run(
```

Comment:
Can this be moved to the `utils` module? Since it's a very general async-handling function.